Expensify/App
# fix: Retain cursor on form submit

@mountiny PR for the cursor issue.

### Details

- Added `blurOnSubmit` for the `PasswordForm`

### Fixed Issues

$ https://github.com/Expensify/App/issues/5191

### Tests
1. Verified focus stays on "Return" and "Enter"
2. Verified that S.ignin button click shouldn't retain the focus


### QA Steps
1. Go to the Password form page
2. Try to enter the password
3. Press Enter and ensure that the input is still focused.



### Tested On


### Screenshots


#### Web
https://user-images.githubusercontent.com/3069065/133500036-645acd3c-c8e8-4b77-8d0e-fa8fe40fb42a.mov

#### Mobile Web

https://user-images.githubusercontent.com/3069065/133499908-fb79ef6e-2c49-4692-86c1-333214cda45e.mov


#### Desktop

https://user-images.githubusercontent.com/3069065/133499747-9c5da634-ccc7-4cb9-9161-261dd49e9094.mov

#### iOS

https://user-images.githubusercontent.com/3069065/133499857-62172868-a4d5-4a5b-a925-f2d06cb8deda.mov


#### Android
https://user-images.githubusercontent.com/3069065/133500261-7f13cdd4-c724-45da-a5cc-a43df6eb53ad.mov
